### PR TITLE
BR: fix backup ranges total time summary log

### LINKED ISSUE #35553: BR: incorrect total summary time for backup ranges
## Bug Report

Please answer these questions before submitting your issue. Thanks!

### 1. Minimal reproduce step (Required)

Run BR, it shows the incorrect total time for backup ranges.

### 2. What did you expect to see? (Required)
Longer total summary time for backup ranges since the total summary time should at least include the backup range finished time.

### 3. What did you see instead (Required)
The total time for backup ranges is quicker than the time taken for finishing backup ranges.
```
[2022/06/16 00:03:22.387 -04:00] [INFO] [client.go:581] ["backup range finished"] [range
-sn=0] [take=682.314138ms]
[2022/06/16 00:03:22.387 -04:00] [INFO] [client.go:567] ["Backup Ranges"] [take=67ns]
```

### 4. What is your TiDB version? (Required)

Current master branch.

### PR DESCRIPTION
**Pull request by ixuh12**
### What problem does this PR solve?

Issue Number: close #35553

Problem Summary: Incorrect total time summary log of backup ranges process
```
[2022/06/16 00:03:22.387 -04:00] [INFO] [client.go:567] ["Backup Ranges"] [take=67ns]
```

### What is changed and how it works?
Fix the code of printing the log. After fixed, it shows the correct time for backup ranges.
```
[2022/06/16 00:17:02.284 -04:00] [INFO] [client.go:538] ["Backup Ranges"] [take=595.572234ms]
```
### Check List

Tests 

- [ ] Unit test
- [ ] Integration test
- [x] Manual test (add detailed scripts or steps below)
- [ ] No code

Side effects

- [ ] Performance regression: Consumes more CPU
- [ ] Performance regression: Consumes more Memory
- [ ] Breaking backward compatibility

Documentation

- [ ] Affects user behaviors
- [ ] Contains syntax changes
- [ ] Contains variable changes
- [ ] Contains experimental features
- [ ] Changes MySQL compatibility

### Release note

Please refer to [Release Notes Language Style Guide](https://pingcap.github.io/tidb-dev-guide/contribute-to-tidb/release-notes-style-guide.html) to write a quality release note.

```release-note
None
```

### CODE REVIEW

**Review by 3pointer — APPROVED** [review]
LGTM

File: br/pkg/backup/client.go [hunk 1]
```diff
@@ -534,7 +534,9 @@ func (bc *Client) BackupRanges(
 	progressCallBack func(ProgressUnit),
 ) error {
 	init := time.Now()
-	defer log.Info("Backup Ranges", zap.Duration("take", time.Since(init)))
+	defer func() {
+		log.Info("Backup Ranges", zap.Duration("take", time.Since(init)))
```
Comment (joccau) [inline br/pkg/backup/client.go:538]: Good ~
Maybe we can print more information about `len(ranges)`.

**Comment by joccau** [comment]
> [CLA assistant check](https://cla.pingcap.net/pingcap/tidb?pullRequest=35552) Thank you for your submission! We really appreciate it. Like many open source projects, we ask that you all sign our [Contributor License Agreement](https://cla.pingcap.net/pingcap/tidb?pullRequest=35552) before we can accept your contribution.**1** out of **2** committers have signed the CLA.✅ hawkingrei❌ ixuh12You have signed the CLA already but the status is still pending? Let us [recheck](https://cla.pingcap.net/check/pingcap/tidb?pullRequest=35552) it.

@ixuh12

File: br/pkg/backup/client.go (hunk 1 above)
Comment (ixuh12) [reply br/pkg/backup/client.go:538]: Thanks! 
And yes, later I will have a follow-up pull request for modifying some printed log messages, which will also include the `len(ranges)` part.

**Review by joccau — APPROVED** [review]
LGTM

**Pull request merged**